packages/@okta/vuepress-theme-prose/components/Footer.vue

```diff
@@ -100,19 +107,19 @@ export default {
   },
   data() {
     return {
-      showMagentaCircle: false,
+      isHomePage: false,
     };
   },
   watch: {
-    $route: "setShowMagentaCircle",
+    $route: "setIsHomepage",
   },
   methods: {
-    setShowMagentaCircle() {
-      this.showMagentaCircle = ["/", "/pricing/"].indexOf(window.location.pathname) >= 0;
```

Comment:
Not sure if this was the result of attempting to resolve a merge conflict, but the code in related to `showMagentaCircle` should not be changed here